cspell.config.yaml

```diff
@@ -65,6 +67,10 @@ words:
   - zstandard
   - zstd
 
+languageSettings:
+  - languageId: "markdown"
+  - "ignoreRegExpList": ["/^\\s*```[\\s\\S]*?^\\s*```/gm"]
```

Comment:
Does this really work? Some of our ignoreRegExpLists don't have slashes around them... are both syntaxes allowed?

Also, I think the `-` at the beginning of this line is wrong. I assume you wanted the ignoreRegExpList to be part of the settings with `languageId: markdown` and not a separate entry in languageSettings?

Reply:
it does work to ignore code blocks, though not sure if it's ignoring other things as well. Good catch with the extra `-`.